packages/grafana-data/src/types/navModel.ts

```diff
@@ -17,6 +17,7 @@ export interface NavModelItem {
   showOrgSwitcher?: boolean;
   onClick?: () => void;
   menuItemType?: NavMenuItemType;
+  suffix?: string;
```

Comment:
this feels basically the same as `highlightText` from https://github.com/grafana/grafana/pull/44048? if so, could we just use that for both? `suffix` kinda means nothing in the context of a navItem. it only really has meaning when talking about the tabs.

Reply:
Good point, haven't thought about that.